adoc/retail_chap_install.adoc

```diff
@@ -148,9 +149,10 @@ zypper in -t pattern suma_retail
 * dhcpd-formula
 * image-sync-formula
 * pxe-formula
+* saltboot-formula
 * susemanager-retail-tools
 * tftp-formula
-* vsftpd-formula
+* vsftpd-formula (optional)
```

Comment:
Please do not specify this as optional. When `retail_branch_init` script is used to configure branch server it expects all formulas to be present.